src/Domains/Inventory/Attributes/DataTransferObject/Attributes.php

```diff
@@ -34,6 +37,7 @@ public static function viaRequest(array $request): self
             auth()->user(),
             $request['name'],
             $request['slug'] ?? Str::slug($request['name']),
+            isset($request['attribute_type']['id']) ? AttributesTypesRepository::getById((int) $request['attribute_type']['id']) : null,
```

Comment:
@arfenis dont you need to make sure the attribute type is from this company and app :?